rfc/rfc004-distributed-document-format.md

```diff
@@ -75,19 +70,19 @@ The following headers MAY be present as protected headers \(see section 3.4 for
 
   defaults to `0`.
 
-To aid performance of validating the DAG the JWS SHALL NOT contain the actual contents of the document. Instead, the JWS payload MUST contain the SHA-1 hash of the contents encoded as hexadecimal, lower case string, e.g.: `148b3f9b46787220b1eeb0fc483776beef0c2b3e`
+To aid performance of validating the DAG the JWS SHALL NOT contain the actual contents of the document. Instead, the JWS payload MUST contain the SHA-256 hash of the contents encoded as hexadecimal, lower case string, e.g.: `386b20eeae8120f1cd68c354f7114f43149f5a7448103372995302e3b379632a`
```

Comment:
did you take a sha256 of something funny?

Reply:
You'll never know, unless SHA-256 is broken (or you perform a dictionary attack of what input I could possibly have used) ;-)

Reply:
😏